docs/reference/UDS Core/IdAM/customization.md

```diff
@@ -14,14 +14,54 @@ See [Testing custom image in UDS Core](https://uds.defenseunicorns.com/reference
 
 Once `uds-core` has sucessfully deployed with your new image, viewing the Keycloak pod can provide insight into a successful deployment or not. Also describing the Keycloak pod, should display your new image being pulled instead of the default image defined [here](https://github.com/defenseunicorns/uds-core/blob/main/src/keycloak/chart/values.yaml#L10) in the events section.
 
+### Branding customizations
+
+The UDS Identity Config supports a limited and opinionated set of branding customizations. This includes:
+
+* Changing the logo
+* Changing the footer image
+* Changing the favicon
+* Changing the background image
+
+These customizations require overriding the Keycloak Helm Chart provided by the UDS Core. Here's an example:
+
+```yaml
+packages:
+  - name: core
+    repository: oci://ghcr.io/defenseunicorns/packages/uds/core
+    ref: x.x.x
+    overrides:
+      keycloak:
+        keycloak:
+          values:
+            - path: themeCustomizations
+              value:
+                 resources:
+                    images:
+                       - name: background.jpg
+                         configmap:
+                            name: keycloak-theme-overrides
+                       - name: logo.svg
+                         configmap:
+                            name: keycloak-theme-overrides
+                       - name: footer.png
+                         configmap:
+                            name: keycloak-theme-overrides
+                       - name: favicon.svg
+                         configmap:
+                            name: keycloak-theme-overrides
+```
+
+The configuration supports only four potential keys: `background.jpg`, `logo.svg`, `footer.png`, and `favicon.svg` which are expected to exist in the corresponding ConfigMaps. The values of these keys are base64 encoded images hosted as `binaryData` part of the ConfigMap. In this example, all four images reside in the same ConfigMap named `keycloak-theme-overrides`.
```

Comment:
Could we also provide an example of how the end user would take their image files and create this configmap? I think it could be confusing for someone new to storing files in configmaps like this.